testproject-tools-integration/Assets/Tests/Runtime/testproject.toolsintegration.runtimetests.asmdef

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "TestProject.ToolsIntegration.RuntimeTests",
+    "rootNamespace": "TestProject.ToolsIntegration.RuntimeTests",
+    "references": [
+        "Unity.Netcode.Runtime",
+        "Unity.Netcode.RuntimeTests",
+        "Unity.Multiplayer.MetricTypes",
+        "Unity.Multiplayer.NetStats"
+    ],
+    "defineConstraints": [
+        "UNITY_INCLUDE_TESTS"
+    ],
+    "optionalUnityReferences": [
+        "TestAssemblies"
+    ],
+    "versionDefines": [
+        {
+            "name": "com.unity.multiplayer.tools",
+            "expression": "",
+            "define": "MULTIPLAYER_TOOLS"
```

Comment:
Under what circumstances is this expected to be false in this project? Or is it just a "shortcut" to define it for non-tools tests?

Reply:
I'm guessing this is just following suit on how we enable this define in other areas. Its a good point though, this define is probably not necessary for this assembly - we could just write the code unconditionally since the whole premise of this project is that tools is installed.

Reply:
I can try defining without the conditional and see if that behaves itself, I had a bit of a nightmare trying to get this to compile and tried many different permutations to get this to work so this may have been unnecessarily left in!

Reply:
Have removed this and it seems to be working OK